akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -1116,7 +1135,7 @@ private[stream] object Collect {
     }
   }
 
-  val NotYetThere = Failure(new Exception)
+  val NotYetThere = Failure(new Exception with NoStackTrace)
```

Comment:
Without a stacktrace this is a very sparse exception... perhaps add a message to it?

Reply:
and shouldn't it be `RuntimeException`?

Reply:
It's never thrown, just used as a marker

Reply:
ok, it's internal I see now